Makefile

```
@@ -1,12 +1,16 @@
 SHELL ?= /bin/bash
 
-ARGO_CD_CHART_VERSION := 5.46.6
+ARGO_CD_CHART_VERSION := 5.46.3
 BUF_LINT_ERROR_FORMAT ?= text
 GO_LINT_ERROR_FORMAT ?= colored-line-number
 CERT_MANAGER_CHART_VERSION := 1.11.5
 
 VERSION_PACKAGE := github.com/akuity/kargo/internal/version
 
+# default docker , change to podman if you using podman
+CONTAINER_RUNTIME = docker
```

Did you test this? This line sets it to `docker` unconditionally. i.e. Whatever is in the env var gets overwritten here.

https://www.gnu.org/software/make/manual/html_node/Setting.html

```suggestion
CONTAINER_RUNTIME ?= docker
```